### DISCUSSION TITLE
Where is BOM 1.16.0?

### DISCUSSION THREAD

**Discussion by nguyennh-0786** [WalletConnect/walletconnect-monorepo#3570] [Developer Support] [answered]
I see BOM 1.16.0 in Readme but do not see it in the release. So, how can I get it?

**Comment by boidushya** [comment]
Hey @nguyennh-0786 could you provide more context about your issue? Which README do you mean specifically?

**Comment by nguyennh-0786** [comment]
@boidushya In here https://github.com/WalletConnect/WalletConnectKotlinV2/blob/develop/ReadMe.md I see BOM v1.16.0 but can not see the lastest release 1.16.0 in https://github.com/WalletConnect/WalletConnectKotlinV2/releases

**Reply by boidushya** [reply, marked as answer]
v1.16.0 is under way, teams merging a few final things. Release should up soon 🤝
